adafruit_requests.py

```diff
@@ -551,6 +558,16 @@ def _send(socket: SocketType, data: bytes):
                 raise OSError(errno.EIO)
             total_sent += sent
 
+    def _send_as_bytes(self, socket: SocketType, data: str):
+        return self._send(socket, bytes(data, "utf-8"))
+
+    def _send_header(self, socket, header, value):
+        self._send_as_bytes(socket, header)
+        self._send_as_bytes(socket, ": ")
+        self._send_as_bytes(socket, value)
+        self._send_as_bytes(socket, "\r\n")
```

Comment:
```suggestion
        self._send_as_bytes(socket, b"\r\n")
```

Reply:
fixed in https://github.com/adafruit/Adafruit_CircuitPython_Requests/pull/149/commits/5fa5aa218bbaecdd7b81c1e33a0894de635a8eff